src/network/networkctl.c

```diff
@@ -1389,87 +1389,126 @@ static int dump_lldp_neighbors(Table *table, const char *prefix, int ifindex) {
         return dump_list(table, prefix, buf);
 }
 
+static int parse_dhcp_lease_client_id(const char *name, JsonVariant *v, JsonDispatchFlags flags, void *userdata) {
+        char **value = ASSERT_PTR(userdata);
+        _cleanup_free_ uint8_t *client_id_bytes = NULL;
+        _cleanup_free_ char *client_id = NULL;
+        size_t client_id_sz;
+        int r;
+
+        r = json_variant_bytes(v, &client_id_bytes, &client_id_sz);
+        if (r < 0)
+                return r;
+        if (!client_id_bytes || client_id_sz == 0)
+                return -EINVAL;
+        r = sd_dhcp_client_id_to_string(client_id_bytes, client_id_sz, &client_id);
```

Comment:
Please insert an empty line before `sd_dhcp_client_id_to_string()`.

Also, please directly pass `value` to `sd_dhcp_client_id_to_string()`.